Framework/API/inc/MantidAPI/DetectorInfo.h

```diff
@@ -138,6 +139,10 @@ class MANTID_API_DLL DetectorInfo {
   detIdToIndexMap() const;
   friend class SpectrumInfo;
 
+  /// Function to find peaks near detector edge
+  static bool edgePixel(Geometry::Instrument_const_sptr inst,
```

Comment:
This should not be added to `DetectorInfo` since it is not compliant with the design of Instrument-2.0, in particular:
- It is using `Geometry::Instrument`.
- It is rather specific, so it is unclear if this should even be available in any form as part of `DetectorInfo`.
- It is pulling an include of `Geometry/Instrument.h` into `DetectorInfo.h`, polluting the includes of all downstream client code using `DetectorInfo`.